### PR TITLE
[docs] platform 5.x

### PR DESCRIPTION
**Pull request by withshubh**
#### changes
- White label doc revamp
- removed community info from copilot doc
- renamed gds to data sources (v2.15 onwards)
- updated multienv doc
- Updated super admin wrt new licensing
- Licensing doc updates

#### additional changes
- #7468

### CODE REVIEW

File: docs/docs/Enterprise/superadmin.md [hunk 1]
```diff
@@ -60,47 +62,54 @@ If a user is Super Admin, they just need to open the workspace in which they wan
...
+- **✅ Copilot**: Enabling or disabling Copilot
+- **✅ Custom Styles**: Adding or modifying custom styles
 
 ## Instance Settings
 
 Only Super Admins can access the Instance Settings:
 
-- **Manage All Users**
+- **All Users**
 - **Manage Instance Settings**
```
[the remark this answers is not in the document]
Comment (withshubh) [reply docs/docs/Enterprise/superadmin.md:88]: I did not remove it. It's rename as per the UI.

File: docs/docs/Enterprise/superadmin.md [hunk 2]
```diff
@@ -17,36 +17,38 @@ The user details entered while setting up ToolJet will have Super Admin privileg
...
 | Manage Workspace Variables in their workspace | ✅ | ✅ |
-| [Manage Global datasources for the user group in their workspace](/docs/data-sources/overview#permissions) | ✅ | ✅ |
+| Manage Workspace Constants in their workspace | ✅ | ✅ |
+| [Manage data sources for the user group in their workspace](/docs/data-sources/overview#permissions) | ✅ | ✅ |
 | [Access any user's personal workspace (create, edit or delete apps)](#access-any-workspace) | ❌ | ✅ |
 | [Archive Admin or any user of any workspace](#archiveunarchive-users) | ❌ | ✅ |
 | [Access any user's ToolJet database (create, edit or delete database)](#access-tooljet-db-in-any-workspace) | ❌ | ✅ |
-| [Manage any workspace's setting (Groups/SSO/Workspace Variables)](#manage-workspace-setting-groupsssoworkspace-variables) | ❌ | ✅ |
-| [Manage all users from all the workspaces in the instance](#checking-all-the-users-in-the-instance) | ❌ | ✅ |
+| [Manage any workspace's setting (Groups/SSO/Workspace constants)](#manage-workspace-setting-groupsssoworkspace-constants) | ❌ | ✅ |
```
[the remark this answers is not in the document]
Comment (withshubh) [reply docs/docs/Enterprise/superadmin.md:25]: Admin's can't access any workspace settings.

File: docs/docs/Enterprise/superadmin.md [hunk 3]
```diff
@@ -17,36 +17,38 @@ The user details entered while setting up ToolJet will have Super Admin privileg
...
+| [Manage all users from all the workspaces in the instance](#manage-all-users-in-the-instance) | ❌ | ✅ |
 | [Make any user Super Admin](#make-the-user-super-admin) | ❌ | ✅ |
 | [Restrict creation of personal workspace of users](#restrict-creation-of-personal-workspace-of-users) | ❌ | ✅ |
+| [Enable multiplayer editing](#enable-multiplayer-editing) | ❌ | ✅ |
```
[the remark this answers is not in the document]
Comment (withshubh) [reply docs/docs/Enterprise/superadmin.md:30]: done

File: docs/docs/Enterprise/superadmin.md [hunk 4]
```diff
@@ -17,36 +17,38 @@ The user details entered while setting up ToolJet will have Super Admin privileg
...
```
[the remark this answers is not in the document]
Comment (withshubh) [reply docs/docs/Enterprise/superadmin.md:29]: done

File: docs/docs/Enterprise/superadmin.md [hunk 5]
```diff
@@ -17,36 +17,38 @@ The user details entered while setting up ToolJet will have Super Admin privileg
...
+<img className="screenshot-full" src="/img/enterprise/superadmin/instanceset.png" alt="Super Admin: Enterprise" />
 
 </div>
 
 ## Super Admin features
 
 ### Access any workspace
 
-If a user is a Super Admin then they can switch to any workspace created by any user in the instance from the dropdown on the top-left of dashboard that is used to switch between workspaces.
+If a user is a Super Admin then they can switch to any workspace created by any user in the instance from the **Workspace Switcher** dropdown in the bottom left corner of the screen.
```
[the remark this answers is not in the document]
Comment (withshubh) [reply docs/docs/Enterprise/superadmin.md:41]: done

File: docs/docs/Enterprise/superadmin.md [hunk 6]
```diff
@@ -60,47 +62,54 @@ If a user is Super Admin, they just need to open the workspace in which they wan
...
 <div style={{textAlign: 'center'}}>
 
-<img className="screenshot-full" src="/img/enterprise/superadmin/instancesettings.png" alt="Super Admin: Enterprise" />
+<img className="screenshot-full" src="/img/enterprise/superadmin/allusersa.png" alt="Super Admin: Enterprise" />
 
 </div>
 
 ### Archiving a user from workspace
 
 Super Admins have the privilege to remove any user from any of the workspace they belong.
```
[the remark this answers is not in the document]
Comment (withshubh) [reply docs/docs/Enterprise/superadmin.md:106]: done

File: docs/docs/Enterprise/white-label.md [hunk 7]
```diff
@@ -5,35 +5,44 @@ title: White Label
 
 <div className='badge badge--primary heading-badge'>Available on: Paid plans</div>
 
-White Label feature will allow you to remove the ToolJet branding from the ToolJet platform and add your own custom logo and text.
+The White Label feature allows you to completely remove ToolJet's branding from the ToolJet platform, allowing you to replace it with your own customized logo and text. This feature grants you full control over the visual identity of your ToolJet-powered application, ensuring a seamless and personalized user experience.
```
[the remark this answers is not in the document]
Comment (withshubh) [reply docs/docs/Enterprise/white-label.md:8]: done

File: docs/docs/Enterprise/white-label.md [hunk 8]
```diff
@@ -5,35 +5,44 @@ title: White Label
...
+</div>
 
 ## Configuration
 
-To enable white labelling, you'll need to set the below mentioned **environment variables** in the .env file:
+To enable white labelling, you'll need to go to the **Instance Settings** from the Dashboard and click on the `White labelling` tab. On the White labelling page, you'll be able to configure the following:
+
+- **Application Logo**: Add the URL of the image you want to use as your application logo. Preferred dimensions of the logo are: width `130px` and height `26px`.
+- **Favicon**: Enter the URL of the image you want to use as your application's favicon. Preferred dimensions of the favicon are: width `32px` and height `32px` or `16px` and height `16px`.
+- **Page Title**: Enter the text you want to display as your application's title.
```
[the remark this answers is not in the document]
Comment (withshubh) [reply docs/docs/Enterprise/white-label.md:42]: done

File: docs/docs/Enterprise/superadmin.md [hunk 9]
```diff
@@ -124,8 +141,17 @@ When a user joins a workspace, they are provided with their own personal workspa
...
-<div style={{textAlign: 'center'}}>
+### Enable multiplayer editing
 
-<img className="screenshot-full" src="/img/enterprise/superadmin/personal.png" alt="Super Admin: Enterprise" />
+Super Admins can enable multiplayer editing from the Manage Instance Settings page. Once enabled, users will be able to edit the same app simultaneously resulting in real-time collaboration.
 
-</div>
+## License
+
+Manage the license of the instance from the Manage Instance Settings page. Super Admins can **update** the license of the instance from this page.
```
[the remark this answers is not in the document]
Comment (withshubh) [reply docs/docs/Enterprise/superadmin.md:150]: done

File: docs/docs/licensing.md [hunk 10]
```diff
@@ -11,13 +11,18 @@ This comprehensive guide aims to assist you in the process of configuring paid p
 
 Let's look into three types of licenses:
 
-1. **Trial License**: This is a free license that grants access to premium features for a 14-day trial period. In order to get this, please reach out to our team. 
+1. **Trial License**: This is a free license that grants access to premium features for a 14-day trial period.
+ - **New Users**: Can opt for free 14-day trial during onboarding.
```
[the remark this answers is not in the document]
Comment (withshubh) [reply docs/docs/licensing.md:15]: done

File: docs/docs/setup/env-vars.md [hunk 11]
```diff
@@ -69,7 +69,7 @@ Use this environment variable to enable/disable the feature that allows you to a
 | ---------------------- | ----------------- |
 | COMMENT_FEATURE_ENABLE | `true` or `false` |
 
-### Multiplayer feature enable ( optional )
+### Multiplayer feature enable ( Only on paid plans )
 
 Use this environment variable to enable/disable the feature that allows users to collaboratively work on the canvas.
```
[the remark this answers is not in the document]
Comment (withshubh) [reply docs/docs/setup/env-vars.md]: done

**Review by gsmithun4 — APPROVED** [review]

**Pull request merged**